banzai_nres/rv.py

```diff
@@ -134,5 +146,15 @@ def do_stage(self, image) -> ObservationFrame:
         image.meta['TCOREPOS'] = 'ERFA', 'Source of Earth position'
         image.meta['TCORSYST'] = 'BJD_TDB ', 'Ref. frame_timesystem of TCORR column'
         image.meta['PLEPHEM'] = solar_system_ephemeris.get(), 'Source of planetary ephemerides'
+        # save the fine + coarse ccfs together
+        coarse_ccfs.sort('order')
+        ccfs.sort('order')
+        final_ccfs = Table({'order': ccfs['order'],
+                            'v': np.hstack([coarse_ccfs['v'], ccfs['v']]),
+                            'xcor': np.hstack([coarse_ccfs['xcor'], ccfs['xcor']])})
```

Comment:
We should insert the fine CCF into the middle of the coarse CCF rather than interleaving them, as the current method could cause problems if there are e.g. different normalizations of the two CCFs.